__tests__/sinopiaServer.test.js

```diff
@@ -27,9 +27,13 @@ describe('sinopiaServerSpoof', () => {
   })
 
   describe('getResourceTemplate', () => {
-    it('known id: returns JSON for resource template', () => {
-      expect(sinopiaServer.getResourceTemplate('resourceTemplate:bf2:Title').id).toEqual('resourceTemplate:bf2:Title')
-      expect(sinopiaServer.getResourceTemplate('resourceTemplate:bf2:Title').resourceLabel).toEqual('Instance Title')
+    it('known id: returns JSON for resource template', async () => {
+      // TODO: This only works with spoofed resource templates. prefix w/
```

Comment:
This TODO is relatively easy to knock out, but doing so—namely by making `sinopiaServer.getSpoofedResourceTemplate()` return a promise—will cause the linkedData reducer unit test to fail. I am **not** inclined to jump head-first into async hell in this issue which is currently blocking other work.